### PR TITLE
SEO-184086-WPF-changes-Page with Redirect

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @gowthamanthavasiyappan,

Worked on page with redirect for wpf.

Regards,
Sharly A.

### CODE REVIEW

File: wpf/Diagram/Constraints.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ ConnectorViewModel connector1 = new ConnectorViewModel()
 {% endhighlight %}
 {% endtabs %} 
 
-![WPF Diagram displays GraphConstraints](Constraints_images/wpf-diagram-group-contraints.gif)
+![WPF Diagram displays graphconstraints.](constraints_images/wpf-diagram-group-contraints.gif)
```
Comment (gowthamanthavasiyappan) [inline wpf/Diagram/Constraints.md:198]: Have you changed the source location? if changed, share the screenshot here.

**Review by gowthamanthavasiyappan — COMMENTED** [review]
Commented

File: wpf/Diagram/Constraints.md (hunk 1 above)
Comment (Sharly-Atieno) [reply wpf/Diagram/Constraints.md:198]: image

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**